etw/cs/traceloggingdynamic/TraceLoggingDynamic.cs

```diff
@@ -836,7 +897,7 @@ public EventBuilder(int initialMetadataBufferSize, int initialDataBufferSize)
         /// </summary>
         public void AddUnicodeString(string name, String value, EventOutType outType = EventOutType.Default, int tag = 0)
         {
-            this.AddScalarMetadata(name, InType.UnicodeString, outType, tag);
+            this.AddMetadata(name, InType.UnicodeString, outType, tag);

```

Comment:
Note: Merged AddScalarMetadata and AddArrayMetadata methods to just AddMetadata.